debezium-testing/debezium-testing-testcontainers/src/main/java/io/debezium/testing/testcontainers/DebeziumContainer.java

```diff
@@ -90,7 +91,9 @@ private static String lazilyRetrieveAndCacheLatestStable() {
     }
 
     public static DebeziumContainer nightly() {
-        return new DebeziumContainer(String.format("%s:%s", DEBEZIUM_CONTAINER, DEBEZIUM_NIGHTLY_TAG));
+        return new DebeziumContainer(String.format("%s:%s", DEBEZIUM_CONTAINER, DEBEZIUM_NIGHTLY_TAG))
+                .withImagePullPolicy(PullPolicy.ageBased(Duration.ofDays(1)));
```

Comment:
This change is unrelated to this specific work; I'd suggest that if this is needed, we do this in a separate PR.

Reply:
No, it is/was not. It was related to test errors when this is not added. A previously fetched old nightly image (in my case from 3.0.0-SNAPSHOT) broke my builds and caused trouble.
Please accept it as part of this PR because of my PTO I can only come back to this in 3 weeks. I am off for the next 2 weeks. Thank you!